spec/events/inventory_aggregate_spec.rb

```diff
@@ -25,8 +25,8 @@
 
     it "should process a donation event" do
       donation = FactoryBot.create(:donation, organization: organization, storage_location: storage_location1)
-      donation.line_items << build(:line_item, quantity: 50, item: item1)
-      donation.line_items << build(:line_item, quantity: 30, item: item2)
+      donation.line_items << build(:line_item, quantity: 50, item: item1, itemizable_type: "Donation", itemizable_id: donation.id)
```

Comment:
Does this work if you just pass in an `itemizable: donation` ? If not, I'd rather update the factory to allow that, because it seems like a pretty obvious way to use it.